packages/wonder-blocks-form/src/components/labeled-text-field.tsx

```diff
@@ -122,6 +120,15 @@ type Props = {
     autoComplete?: string;
 };
 
+type OtherInputProps = CommonProps & {
+    /**
+     * Determines the type of input. Defaults to text.
+     */
+    type?: "text" | "password" | "email" | "tel";
```

Comment:
Given we have this in the `DefaultProps`, I think we can mark this as non-optional.  See `disabled` prop above which also has a default. 

```suggestion
    type: "text" | "password" | "email" | "tel";
```

Reply:
Is it better to do it this way?

I've always found it odd to make a prop non-optional if it has a default prop, because I think it inherently becomes optional for a user to pass in that prop if it has a default value... right?